test/e2e/fixture/scylla/basic.scyllacluster.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     - name: us-east-1a
       members: 1
       storage:
-        capacity: 100Mi
+        capacity: 300Mi
```

Comment:
just for the record, I recall there were some implications around xfs min size - can you comment on the reason for the bump?

Reply:
XFS min size is related to entire volume quota can be even a single block.

Reply:
>can you comment on the reason for the bump? 

It's part of PR description